### PR TITLE
Allows janibelts to hold trashbags

### PR DESCRIPTION
**Pull request by yyzsong**
fuck you llol

## About The Pull Request
Allows janibelts to hold trashbags

## Why It's Good For The Game
 
image
Ask this guy

## Changelog
:cl:
tweak: the janibelt can now hold trashbags
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -607,6 +607,7 @@
 		/obj/item/assembly/mousetrap,
 		/obj/item/paint/paint_remover,
 		/obj/item/pushbroom
```
Comment (BrumalLake) [inline code/game/objects/items/storage/belt.dm]:
```suggestion
		/obj/item/pushbroom,
```

Comment (yyzsong) [reply code/game/objects/items/storage/belt.dm]: *scream

**Review by ike709 — APPROVED** [review]

**Comment by ike709** [comment]
Actually to prevent people from cheesing storage you should probably limit it to one (1) trash bag.

**Comment by Archanial** [comment]
It makes little sense for janibelts to hold trashbags. 

If you really want this to happen make sure trashbags are empty and exclude bluespace variant.

**Comment by BrumalLake** [comment]
janicart has a purpose along with biosuit storage
multibelt/assault belt on steroids

**Comment by utilbird** [comment]
> Actually to prevent people from cheesing storage you should probably limit it to one (1) trash bag.

Boxes are 100x better for that so it doesn't really matter

**Comment by BrumalLake** [comment]
the trash bag holds more small items and you could fit 6 or 7 on them in a normal sized belt not even counting bluespace

**Comment by utilbird** [comment]
I thought trashbags can only hold tiny items

**Pull request closed without merging**